docs/reference/quickstart/getting-started.asciidoc

```diff
@@ -193,31 +255,216 @@ You should receive a response indicating there were no errors.
+When an index has the `dynamic` flag set to `true`, you can add new fields to documents without updating the mapping.
+This allows you to <<mapping-combine-explicit-dynamic,combine explicit and dynamic mappings>>.
 
 [discrete]
-[[search-all-documents]]
+[[getting-started-search-data]]
+=== Step 4: Search your index
+
+Indexed documents are available for search in near real-time, using the <<search-your-data,`_search` API>>.
+// TODO: You'll find more detailed quick start guides in TODO
+
+[discrete]
+[[getting-started-search-all-documents]]
 ==== Search all documents
 
 Run the following command to search the `books` index for all documents:
+
 [source,console]
 ----
 GET books/_search
 ----
 // TEST[continued]
 
-The `_source` of each hit contains the original
-JSON object submitted during indexing.
+.Expand to see (truncated) example response with annotations
+[%collapsible]
+===============
+[source,console-result]
+----
+{
+  "took": 2, <1>
+  "timed_out": false, <2>
+  "_shards": { <3>
+    "total": 5,
+    "successful": 5,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": { <4>
+    "total": { <5>
+      "value": 7,
+      "relation": "eq"
+    },
+    "max_score": 1, <6>
+    "hits": [
+      {
+        "_index": "books", <7>
+        "_id": "CwICQpIBO6vvGGiC_3Ls", <8>
+        "_score": 1, <9>
+        "_source": { <10>
+          "name": "Brave New World",
+          "author": "Aldous Huxley",
+          "release_date": "1932-06-01",
+          "page_count": 268
+        }
+      },
+      ... (truncated)
+    ]
+  }
+}
+----
+// TEST[continued]
+<1> The `took` field indicates the time in milliseconds for {es} to execute the search
+<2> The `timed_out` field indicates whether the search timed out
+<3> The `_shards` field contains information about the number of <<glossary,shards>> that the search was executed on and the number that succeeded
+<4> The `hits` object contains the search results
+<5> The `total` object provides information about the total number of matching documents
+<6> The `max_score` field indicates the highest relevance score among all matching documents
+<7> The `_index` field indicates the index the document belongs to
+<8> The `_id` field is the document's unique identifier
+<9> The `_score` field indicates the relevance score of the document
+<10> The `_source` field contains the original JSON object submitted during indexing
+===============
 
 [discrete]
-[[qs-match-query]]
+[[getting-started-match-query]]
 ==== `match` query
 
 You can use the <<query-dsl-match-query,`match` query>> to search for documents that contain a specific value in a specific field.
-This is the standard query for performing full-text search, including fuzzy matching and phrase searches.
+This is the standard query for full-text searches.
 
 Run the following command to search the `books` index for documents containing `brave` in the `name` field:
```

Comment:
nit: we didn't get them to add this book in the tutorial.

Reply:
huh? Brave New World? What am I missing?

Reply:
if someone is following this tutorial from top to bottom, they'd currently only have two books in their index: snow crash and the great gatsby

so the search would return no results

Reply:
What about the bulk api call? You've added multiple additional documents there

Reply:
https://elasticsearch_bk_113558.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/getting-started.html#getting-started-add-multiple-documents

Reply:
hi im illiterate this pr is perfect :hide: